drivers/usb/udc/udc_rpi_pico.c

```diff
@@ -1027,8 +1028,11 @@ static int udc_rpi_pico_enable(const struct device *dev)
 	usb_hw_t *base = config->base;
 
 	/* Reset USB controller */
-	reset_block(RESETS_RESET_USBCTRL_BITS);
-	unreset_block_wait(RESETS_RESET_USBCTRL_BITS);
+	int ret = reset_line_toggle_dt(&config->reset);
+
+	if (ret < 0) {
+		return ret;
+	}
```

Comment:
Since you are not a first-time contributor and have already submitted patches and received comments, especially on this topic, it does not seem like you are putting in much effort. Please be kind and follow the contribution guidelines, coding style, and style in the subsystem/file, place variable definitions at the top of a block. Also, please avoid introducing undefined behavior when checking return variables. Thanks.

```c
	usb_hw_t *base = config->base;
	int ret;

	ret = reset_line_toggle_dt(&config->reset);
	if (ret) {
		return ret;
	}
```